SocialCareCaseViewerApi/V1/Infrastructure/SccvDbContext.cs

```diff
@@ -25,22 +25,22 @@ public SccvDbContext()
             X509Store localTrustStore = new X509Store(StoreName.Root);
             //  string caContentString = System.IO.File.ReadAllText(cAfile);
 
-            X509Certificate2 caCert = new X509Certificate2(Encoding.ASCII.GetBytes(cAfile));
+            // X509Certificate2 caCert = new X509Certificate2(Encoding.ASCII.GetBytes(cAfile));
```

Comment:
please enable the commented out lines here

Reply:
Every time!